v1/examples/pipeline/steps/service-now-approval-step.yaml

```diff
@@ -0,0 +1,26 @@
+version: 1
+kind: pipeline
+spec:
+  stages:
+    - type: approval
+      name: approval
+      id: Approval
+      timeout: 10m
+      when: <+<+environment.name> != "QA">
+      spec: 
+        steps:
+          - type: service-now-approval
+            name: serviceNow
+            id: serviceNow
+            spec: 
+              connector: SNOWConnector
+              number: "123"
+              type: change_request
```

Comment:
this should be `change-request`

Reply:
We are getting the response from Service Now ticket type API, shall we change this?